File: Python/Palindrome_Number.py
def isPalindrome(num):
	tmp=num
	res=0
	while(num>0):
	    dig=num%10
	    res=res*10+dig
	    num=num//10
	if(tmp==res):
	    print("The number is palindrome!")
	else:
	    print("Not a palindrome!")

isPalindrome(12321)

File: C++/palindrome_number.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
	int num;
	cout<<"Enter a number \n";
	cin>>num;
    int rev=0,val;
    val = num;
    while(num > 0){
      rev = rev * 10 + num % 10;
      num = num / 10;
    }
    if(val==rev)
    cout<<val<<" is a palindrome"<<endl;
    else
    cout<<val<<" is not a palindrome"<<endl;
    return 0;
}
